osu.Game/Rulesets/Mods/UnknownMod.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Mods
+{
+    public class UnknownMod : Mod
```

Comment:
It's a bit of a nitpick as it won't ever realistically matter but I'd also override `UserPlayable` to `false` here, just for consistency I guess given that it's part of the `IMod` interface.